cedar-lean/Cedar/SymCC/Encoder.lean

```diff
@@ -147,7 +147,7 @@ https://smtlib.cs.uiowa.edu/theories-UnicodeStrings.shtml. Here,
 we're assuming ASCII strings for simplicity.
 -/
 def encodeString (s : String) : String :=
-  s!"\"{s}\""
+  s!"\"{s.replace "\"" "\"\""}\""
```

Comment:
perhaps adding a comment here about `"` being the only character that needs escaping in SMTLib?